Sources/SATSCore/BasicComponents/SATSCell/SATSCell-BasicCell.swift

```diff
@@ -0,0 +1,76 @@
+import SwiftUI
+
+public extension SATSCell {
+    /// A button to implement a custom table-view cell
+    struct BasicCell<IconView: View, AccessoryView: View, ActionView: View>: View {
+        @ViewBuilder var icon: IconView
+        var title: String
+        var subtitle: String?
+        @ViewBuilder var accessory: AccessoryView
+        @ViewBuilder var action: ActionView
+        var includeDivider: Bool = true
+        var onClick: (() -> Void)?
+
+        public init(
+            @ViewBuilder icon: () -> IconView = SATSCell.NoIcon.init,
+            title: String,
+            subtitle: String? = nil,
+            @ViewBuilder accessory: () -> AccessoryView = EmptyView.init,
+            @ViewBuilder action: () -> ActionView = SATSCell.SystemChevron.init,
+            includeDivider: Bool = true,
+            onClick: (() -> Void)? = nil
+        ) {
+            self.icon = icon()
+            self.title = title
+            self.subtitle = subtitle
+            self.accessory = accessory()
+            self.action = action()
+            self.includeDivider = includeDivider
+            self.onClick = onClick
+        }
+
+        public var body: some View {
+            Button(action: { onClick?() }) {
+                HStack(alignment: .center, spacing: .spacingS) {
+                    icon
+                        .padding(.leading, .spacingS)
+
+                    VStack(spacing: 0) {
+                        HStack(spacing: .spacingXS) {
+                            VStack(alignment: .leading, spacing: .spacingXXS) {
+                                Text(title)
+                                    .satsFont(.basic)
+
+                                subtitleLabel
+                            }
+                            Spacer()
+                            accessory
+                            action
+                        }
+                        .padding(.trailing, .spacingS)
```

Comment:
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/9019310/214234511-7e862878-f619-4ac6-86c4-eb452cb59e4c.png">


should be .spacingM

Reply:
good catch!